stack.yaml

```diff
@@ -15,4 +15,5 @@ extra-deps:
 - libsystemd-journal-1.4.4
 
 nix:
-  shell-file: nix/stack-shell.nix
+  enable: true
```

Comment:
This will not work well for nix non-users. The best place for this is in `~/.stack/config.yaml`.